docs/checkstyle/check-docs.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -ue
+
+ret=0
+cd docs/guides
+
+for docs in admin developer; do
+  cd $docs
+  echo "Markdown doc $docs build log:"
+  mkdocs build &> mkdocs.log
+  cat mkdocs.log
```

Comment:
Would it make sense to move this line into the `if` so that the log isn't always printed?